Modules/@babylonjs/react-native/windows/BabylonReactNative/EngineView.cpp

```diff
@@ -66,6 +67,14 @@ namespace winrt::BabylonReactNative::implementation {
         const auto position = point.Position();
         const uint32_t x = position.X < 0 ? 0 : static_cast<uint32_t>(position.X);
         const uint32_t y = position.Y < 0 ? 0 : static_cast<uint32_t>(position.Y);
+        const auto pointerId = point.PointerId();
+
+        if (!_inputSource.HasCapture())
+        {
+            _inputSource.SetPointerCapture();

```

Comment:
So an input source as a whole has capture, rather than each pointer? For touch input for example, capture applies to all your fingers, not each individual one?

Reply:
yes. it seems to work differently for CoreIndependentInputSource vs xaml pointers. The pointers obtained here can't be individually captured